_rules/SC3-1-2-lang.md

```diff
@@ -1,79 +1,78 @@
 ---
-rule_id: SC3-1-2-lang
-name: Languages within the body
-test_mode: automatic
-environment: DOM Structure
+name: Validity of lang attribute value in html body elements
```

Comment:
This name is too long. Names should be two or three words long.

Reply:
How about one of the below?
- Valid `lang` for body elements
- Body Elements `lang` validity
- Valid `lang` within body

Reply:
@Brynanders - any better suggestions here?